compiler/crates/relay-lsp/src/goto_definition/mod.rs

```diff
@@ -79,12 +79,13 @@ pub fn on_goto_definition(
     let program = state.get_program(&project_name)?;
 
     let definition_description = match feature {
-        crate::Feature::GraphQLDocument(document) => {
+        crate::Feature::ExecutableDocument(document) => {
             get_graphql_definition_description(document, position_span, &schema)?
         }
         crate::Feature::DocblockIr(docblock_ir) => {
             get_docblock_definition_description(&docblock_ir, position_span)?
         }
+        crate::Feature::SchemaDocument(_) => todo!(),
```

Comment:
I think this `todo!()` will make the LSP panic and quit. Should we put something less destructive here? Or should be plan to land this atomically with the other PRs you also opened?

Reply:
Ah sorry, I meant to replace this with an expected error. One sec.

Reply:
Replaced!